docs/scripts/generate_plugin_docs.py

```
@@ -346,6 +351,152 @@ def fetch_github_stars(self, owner: str, repo: str) -> Optional[dict]:
+
+            display_tags = self._format_model_tags(tags)
+            display_tags.append("Plugin")
+
+            self._plugin_model_cards.append(f"""
+.. customcarditem::
+    :header: {name}
+    :description: {description}
+    :link: models/{model_slug}.html
+    :tags: {",".join(display_tags)}
+""")
+            logger.info(f"Generated model docs for {name}")
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)